elements/_base.scss

```diff
@@ -0,0 +1,28 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: $font-open-sans;
+    
+}
+
+a {
+    text-decoration: none;
+}
+
+h2 {
+    font-family: $font-raleway;
+    font-size: 2.25rem;
+}
+
+
+.container {
+    width: 1352px;
+    margin: 0 auto;
+
+    @include mobile {
+        width: 100%;
+        padding: 0 15px;
+    }
+}
```

Comment:
ta klasa nie pasuje do tego pliku. w powyzszych przykladach stylujesz tagi html, restartujesz je jakoby, container to klasa. juz lepiej dac ja do main.scss jak nigdzie nie pasuje